calc/policy_sorter.go

```diff
@@ -112,6 +113,20 @@ func (p PolKV) String() string {
 	return fmt.Sprintf("%s(%s)", p.Key.Name, orderStr)
 }
 
+func (p *PolKV) GovernsApplyOnForward() bool {
```

Comment:
Naming: `GovernsApplyOnForward` doesn't read so well.  In this context, "Governs" means "Has control over".  "Has control over ingress (traffic)" makes sense but "Has control over apply on forward" doesn't.  

How about `AppliesOnForward` or `GovernsForwardedTraffic`?